RecoTracker/MeasurementDet/python/MeasurementTrackerESProducer_cfi.py

```diff
@@ -5,7 +5,14 @@
 MeasurementTracker = _MeasurementTrackerESProducer_default.clone()
 
 from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
-trackingPhase2PU140.toModify(MeasurementTracker, Phase2StripCPE = 'Phase2StripCPE')
+trackingPhase2PU140.toModify(MeasurementTracker, 
+                             Phase2StripCPE = cms.string('Phase2StripCPE'),
+                             #StripCPE = cms.string(''),
+                             UseStripModuleQualityDB = cms.bool(False),
+                             UseStripAPVFiberQualityDB = cms.bool(False),
+                             MaskBadAPVFibers = cms.bool(False),
+                             UseStripStripQualityDB = cms.bool(False),
```

Comment:
could the explicit types be avoided here?
>When one customizes an existing parameter in clone(), Modifier.toModify(), or in assignment, explicit types on the right hand side should be avoided.

Reply:
done in https://github.com/cms-sw/cmssw/pull/36983/commits/96f4fb5246d84695f0bca88098676c1587d3f916